Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveGridView.Properties.cs

```diff
@@ -94,7 +88,8 @@ private static void OnOneRowModeEnabledChanged(DependencyObject d, object newVal
                     };
 
                     self._listView.SetBinding(GridView.MaxHeightProperty, b);
-                    self.VerticalScrollMode = ScrollMode.Disabled;
+                    ScrollViewer.SetVerticalScrollMode(self, ScrollMode.Disabled);
+                    ScrollViewer.SetVerticalScrollBarVisibility(self, ScrollBarVisibility.Hidden);
```

Comment:
Sorry, for whatever reason I thought these were being set in a constructor :) Ignore this comment then.